src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -290,28 +291,34 @@ const FixPhoneNumbers: React.FC<Props> = ({
                       'Choose below which phone number will be set as primary.',
                     )}
                   </Typography>
-                  <Box className={classes.defaultBox}>
-                    <Typography>{t('Default Primary Source:')}</Typography>
 
-                    <Select
-                      className={classes.select}
-                      inputProps={{ 'data-testid': 'source-select' }}
-                      value={defaultSource}
-                      onChange={(event: SelectChangeEvent) =>
-                        handleSourceChange(event)
-                      }
-                      size="small"
-                    >
-                      {sourceOptions.map((source) => (
-                        <MenuItem
-                          key={source}
-                          value={source}
-                          data-testid="select-option"
-                        >
-                          {source}
-                        </MenuItem>
-                      ))}
-                    </Select>
+                  <Box className={classes.defaultBox}>
+                    <FormControl size="small">
+                      <InputLabel shrink id="source-select" size="small">
```

Comment:
Thanks for making this a `FormControl`! Looks like two elements have the id `source-select` and ids are supposed to be unique. You can probably remove them both unless you know of a reason to keep them. Also, I don't think that `inputProps={{ 'data-testid': 'source-select' }}` is needed anymore since we are finding the select input by its role.